src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Color[] COLORS = Color.values();
+    private static Random random = new Random();
+
+    ColorSupplier() {
+    }
+
+    public static String getRandomColor() {
```

Comment:
it should not be static
```suggestion
    public String getRandomColor() {
```